tests/test_preconf.py

```diff
@@ -148,6 +150,7 @@ def everythings(
         Everything.AnIntEnum.A,
         Everything.AStringEnum.A,
         draw(dts),
+        draw(dates()),
```

Comment:
Please set the max value here similar to what the datetime strategy is doing above. If we don't do this, we might break some 32-bit Linux downstreams (don't ask...).

Reply:
Ah okay 😬  done